### ISSUE TITLE
How add text on a map?

### ISSUE THREAD

**Issue by Cristian-wp** [umap-project/umap#2335]
Hello, I have imported in umap a building geojson with success.
Now I would like to add the rooms name in another layer without using a marker or maybe using an invisible one...is this possible?
I know that I can make the popup always visible, but I is not what I would like to do.

**Comment by yohanboniface** [comment]
Have you considered using a tooltip on a polygon ?

![Image](https://github.com/user-attachments/assets/d56d0261-49b1-4283-8852-43115705488a)

**Comment by Cristian-wp** [comment]
Yes, it can be another solution, but like using markers is a popup, and it will stay the same size at all lelvels of zoom. 
Zoom level 4:
![Image](https://github.com/user-attachments/assets/9f8b317d-2562-46ab-85c3-6b8a829a1fc7)

Zoom level 24:
![Image](https://github.com/user-attachments/assets/a9365704-94e8-42ff-9b3e-b20cd022f335)

If is possible I would like to add directly some texts without a popup. I can not add text directly from the geojson because I will lost the search feature.

**Comment by Elaws** [comment]
Any news please ?
